commands/user.go

```diff
@@ -359,21 +360,27 @@ Global Flags:
 }
 
 func userActivateCmdF(c client.Client, command *cobra.Command, args []string) error {
-	changeUsersActiveStatus(c, args, true)
+	if err := changeUsersActiveStatus(c, args, true); err != nil {
```

Comment:
This can be written as
```suggestion
	return changeUsersActiveStatus(c, args, true)
```

Reply:
Done, thanks for the suggestion 🚀